core/dbt/main.py

```diff
@@ -1084,6 +1084,27 @@ def parse_args(args, cls=DBTArgumentParser):
         """,
     )
 
+    schema_cache_flag = p.add_mutually_exclusive_group()
+    schema_cache_flag.add_argument(
```

Comment:
I am fairly new to all of the command arguments, would like to hear why you chose to add two arguments that would change the same `dest` vs the default behavior is not use this feature and a just one flag to enable the feature.

Reply:
One could set `selected_schema_cache: true` in profile.yml as general config for their project but may choose to override that with `--no-selected-schema-cache` in cli for specific scenario.

Reply:
Right on! The inheritance order for "global" configs is CLI flag > env var (`DBT_{config}`) > profile config

I would vote for naming this config in a way that's slightly more generic (not every database calls it "schema," not every database caches at the schema level). Perhaps `CACHE_SELECTED_ONLY`?